packages/playground/windows/playground-composition/Playground-Composition.cpp

```diff
@@ -707,8 +692,7 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
 #ifdef USE_EXPERIMENTAL_WINUI3
   // Island-support: Create our custom Xaml App object. This is needed to properly use the controls and metadata
   // in Microsoft.ui.xaml.controls.dll.
-  auto playgroundApp{winrt::make<winrt::Playground::implementation::App>()};
-#endif
+  // auto playgroundApp{winrt::make<winrt::Playground::implementation::App>()};
```

Comment:
Is this temporarily commented out?

Reply:
It caused a runtime error trying to create the XamlApplication object.  I commented this out for now so that I could use USE_EXPERIMENTAL_WINUI3 to verify the content island hosting component.

Someone will have to look into how to properly get Xaml working within a fabric application.